src/test/java/com/shopify/ShopifySdkTest.java

```diff
@@ -30,6 +30,7 @@
 import org.junit.Rule;
```

Comment:
I commented on the service but we need to ensure we have tests that assert on created fulfillment orders. It's not strong enough to rely purely on the fulfillments anymore since we'll be calling more APIs.